lizmap/resources/ui/ui_lizmap.ui

```diff
@@ -4771,23 +4781,23 @@ This is different to the map maximum extent (defined in QGIS project properties,
   <customwidget>
    <class>QgsCollapsibleGroupBox</class>
    <extends>QGroupBox</extends>
-   <header>qgis.gui</header>
+   <header>qgscollapsiblegroupbox.h</header>
```

Comment:
I'm surprised test are passing. There should be a test for this one, I will check.
Can you revert these changes ? (and other lines below)

Reply:
done